### DISCUSSION TITLE
What does "Music directories" do ?

### DISCUSSION THREAD

**Discussion by Rexadev** [hyperblast/beefweb#271] [Q&A]
![image](https://github.com/hyperblast/beefweb/assets/62152714/b6cf1df1-edad-4627-bfb4-f9f09013ddc6)

**Comment by deckerth** [comment]
I think that Music directories are required when you want to add new items to playlists via the web interface.

**Comment by Dammouz** [comment]
Hello,
I confirm, it is the list of the several "root" directories where your music files are stored.
A sample is given in the beefweb readme.md file:
Linux:
![Configuration screen for DeaDBeeF](https://user-images.githubusercontent.com/19171756/34526667-40ce832a-f0b4-11e7-8918-16180b6a66ad.png)
Windows:
![Configuration screen for foobar2000](https://user-images.githubusercontent.com/19171756/44335583-9c015380-a47d-11e8-9934-639503538f8e.png)
